Cargo.toml

```diff
@@ -30,11 +30,11 @@ doctest = false # and no doc tests
 required-features = ["rustc_tests"]
 
 [dependencies]
-byteorder = { version = "1.1", features = ["i128"] }
 cargo_metadata = { version = "0.8", optional = true }
 directories = { version = "2.0", optional = true }
 rustc_version = { version = "0.2.3", optional = true }
 getrandom = { version = "0.1.8", features = ["std"] }
+byteorder = "1.3"
```

Comment:
In terms of versions, isn't this a NOP? 1.3 is semver-compatible with 1.1.

Also it is possible that we had to add the feature to make sure we use the exact same feature as other projects in the rustc workspace. So I am not sure that we can just remove it here...